tests/regression/lp-1899664/task.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Test that read-only filesystem on /etc/dbus-1/session.d doesn't prevent snapd refresh.
+systems: [ubuntu-core-18-64]
+
+prepare: |
+  mount -t tmpfs tmptfs -o ro /etc/dbus-1/session.d
+
+execute: |
+  # we are running current snapd build, re-install it to trigger core18
+  # wrappers to be recreated.
+  snap install --dangerous /var/lib/snapd/seed/snaps/snapd_x1.snap 
```

Comment:
Could you revert as well, so that we don't have to worry about the change in revision. I think some environments are not restoring the snapd snap automatically. Please add a comment about this if you can.